src/client/ds3/bucket.c

```diff
@@ -252,9 +251,16 @@ ds3_bucket_get_info(struct ds3_bucket_info *info, ds3_bucket_t *ds3b, daos_event
 	if (rc != 0)
 		return -rc;
 
-	rc = daos_cont_get_attr(coh, 1, names, values, sizes, ev);
-	rc = daos_der2errno(rc);
-	dfs_cont_put(ds3b->dfs, coh);
+	rc = daos_cont_get_attr(coh, 1, names, &info->encoded, &info->encoded_length, ev);
+	if (rc) {
+		rc = daos_der2errno(rc);
+		goto out_put;
```

Comment:
Do not think it is needed ;)

Reply:
sure, but just to be safe if someone else adds something to be function to not introduce a bug.

Reply:
Yep, make sense :)